src/main/scala/is/hail/annotations/UnsafeOrdering.scala

```diff
@@ -1,8 +1,15 @@
 package is.hail.annotations
 
+import is.hail.utils._
+import is.hail.asm4s._
+
 abstract class UnsafeOrdering extends Ordering[RegionValue] with Serializable {
   def compare(r1: MemoryBuffer, o1: Long, r2: MemoryBuffer, o2: Long): Int
 
+  def compare(r1: Code[MemoryBuffer], o1: Code[Long], r2: Code[MemoryBuffer], o2: Code[Long]): Code[Int] =
+    // FIXME
```

Comment:
oh haha, I never actually implemented this part. I'll do that now, @patrick-schultz, my bad!